apps/guardian-ui/src/languages/en.json

```diff
@@ -17,7 +17,7 @@
   "federation-dashboard": {
     "placeholder-fed-description": "A community-based coalition focused on ushering the world into a bitcoin standard",
     "invite-members": "Invite members",
-    "invite-members-prompt": "Share this to invite members to join the federation",
+    "invite-members-prompt": "Share this to invite members to join the federation. Or have them scan the QR code ->",
```

Comment:
One thing to consider with the `->` is that with text wrapping / layout changes in different languages or smaller screen widths, the QR code may not always be directly to the right of this copy. As @Kodylow noted in Discord, it might be good to have it very clear what this QR code is for inline.